Lesson07/HW07.StringWorkerThree/Program.cs

```diff
@@ -0,0 +1,31 @@
+﻿using System;
+using System.Linq;
+
+namespace HW07.StringWorkerThree
+{
+    internal class Program
+    {
+        static void Main(string[] args)
+        {          
+            string initialВata = "gdfgdf234dg54gf+23oP42";
+            Console.WriteLine("Изначальное значение: " + initialВata);
```

Comment:
Bata? )))) and write on english plz. I dont understand...

Reply:
Data! ) ... Done